octoprint_octorelay/driver.py

```diff
@@ -1,21 +1,27 @@
 # -*- coding: utf-8 -*-
 from typing import Optional
-from RPi import GPIO
+from gpiozero.pins.lgpio import LGPIOFactory
+from gpiozero import LED
 
-# The driver operates BCM mode of pins enumeration
-GPIO.setmode(GPIO.BCM)
-
-def xor(left: bool, right: bool) -> bool:
-    return left is not right
 
 class Relay():
-    def __init__(self, pin: int, inverted: bool):
+    relays = []
```

Comment:
consider `cache`

Reply:
I don't think you can move the list inside the init, From what I know inside an object you can't create a new object, that's why I had done the class method. 

I created the `get_or_create` method because if you try to create multiple LED instances with the same pin, gpiozero raises an exception. There is a possibility to avoid this by using gpiozero's `.close()` function, but this resets the pin state to its initial state.

Reply:
I meant the name of the property. So it could be `Relay.cache` instead of `Relay.relays`

Reply:
But it's fine as it for now